core/app/modules/kit-library/assets/js/pages/index/index-no-results.js

```diff
@@ -1,8 +1,41 @@
 import { Heading, Text, Grid, Button } from '@elementor/app-ui';
+import { useMemo } from 'react';
+import { useNavigate, useLocation } from '@reach/router';
 
 import './index-no-results.scss';
 
+function useNoResultContent( isFilterActive, clearFilter ) {
```

Comment:
IDK what's the standard here, but shouldn't it be in an external file?
( Well... you probably won't need that in other places, so it might be the right place )

Reply:
I just create a function to make sure the component itself is clean. and the logic of the content comes from another function. and because the code itself is necessary only for the current component I put it in the same file.

Not sure what is the standard but it looks OK to me. let me know if you find anything else.